.github/workflows/gmc-helm.yaml

```diff
@@ -56,14 +56,30 @@ jobs:
       - name: Install GMC
         run: |
           echo "should_cleanup=true" >> $GITHUB_ENV
-          ## IMPORTANT NOTICE: need to replace gmc images with {{env.DOCKER_REGISTRY}} and {{env.VERSION}}
-          cd microservices-connector
-          if ! helm install --create-namespace --namespace {{ env.SYSTEM_NAMESPACE}} --wait \
+          USER_ID=$(whoami)
+          HFTOKEN=$(cat /home/$USER_ID/.cache/huggingface/token)
+          pushd microservices-connector/helm
+          # replace image for gmc-router
+          sed -i "s|image:.*|image: ${DOCKER_REGISTRY}opea/gmcrouter:$VERSION|" gmc-router.yaml
+          # replace the pull policy "IfNotPresent" with "Always" for gmc-router
+          sed -i "s|imagePullPolicy:.*|imagePullPolicy: Always|" gmc-router.yaml
+          # replace image for GenAI component manifests
+          find manifests_common/ -type f -name "*.yaml" -exec sed -i "s|image: \"opea|image: \"${DOCKER_REGISTRY}opea|" {} \;
+          # replace the pull policy "IfNotPresent" with "Always" for GenAI component manifests
+          find manifests_common/ -type f -name "*.yaml" -exec sed -i "s|imagePullPolicy: IfNotPresent|imagePullPolicy: Always|" {} \;
+          # replace the mount dir "path: /mnt/model" with "path: $CHART_MOUNT"
+          find manifests_common/ -type f -name "*.yaml" -exec sed -i "s|path: /mnt/opea-models|path: $KIND_MOUNT_DIR|" {} \;
+          # replace huggingface token
+          find manifests_common/ -type f -name '*.yaml' -exec sed -i "s#insert-your-huggingface-token-here#${HFTOKEN}#g" {} \;
```

Comment:
also consider google API key?

Reply:
google_api_key and google_cse_id is exposed as secret. To use that, we need to use a different set of event 'pull_request_target' and need quite a lots of change, also using . I think we should fix this along with go e2e CI in another PR, need @daisy-ycguo 's input here

Reply:
Yes. Better to use two PRs. Using secrets is more complex for CI workflows.